docs/development/releasing.md

```diff
@@ -51,32 +51,32 @@ Command: `make release`
 - Try to build it from the release branch, since we include that in the binary version
 - Verify the ldflags output, in particular check the Version, BuildUser, and GoVersion are expected
 
-Once the build is complete, check the VERSION and note the sha256 hash.
-
-## 4. Push the Docker images
-
-`make release` should output a command to push the image.  Alternatively, run:
+Once the build is complete, copy the output after `Release info...` and save it to use in step 5
 
+Example:
 ```
-$ PATCH_VERSION=v0.23.0
-$ docker push gcr.io/cadvisor/cadvisor:$PATCH_VERSION
+Multi Arch Container:
+gcr.io/cadvisor/cadvisor:v0.44.1-test-8
+
+Architecture Specific Containers:
+gcr.io/cadvisor/cadvisor-arm:v0.44.1-test-8
+gcr.io/cadvisor/cadvisor-arm64:v0.44.1-test-8
+gcr.io/cadvisor/cadvisor-amd64:v0.44.1-test-8
+
+Binaries:
+SHA256 (cadvisor-v0.44.1-test-8-linux-arm64) = e5e3f9e72208bc6a5ef8b837473f6c12877ace946e6f180bce8d81edadf66767
+SHA256 (cadvisor-v0.44.1-test-8-linux-arm) = 7d714e495a4f50d9cc374bd5e6b5c6922ffa40ff1cc7244f2308f7d351c4ccea
+SHA256 (cadvisor-v0.44.1-test-8-linux-amd64) = ea95c5a6db8eecb47379715c0ca260a8a8d1522971fd3736f80006c7f6cc9466
 ```
 
-## 5. Cut the release
+## 4. Cut the release
```

Comment:
Should I add running `check_container.sh` as a step to confirm that you can start all of the containers that were generated?

Reply:
That's a good idea, let's add it as validation step

Reply:
Added the validation step